pages/asru-assignment/index.js

```diff
@@ -0,0 +1,82 @@
+const { page } = require('@asl/service/ui');
+const { buildModel } = require('@asl/pages/lib/utils');
+const form = require('@asl/pages/pages/common/routers/form');
+const { schema, getSchemaWithInspectors, getSchemaWithSpocs } = require('./schema');
+
+module.exports = settings => {
+  const app = page({
+    root: __dirname,
+    ...settings
+  });
+
+  app.use((req, res, next) => {
+    req.model = buildModel(schema);
+    req.model.id = 'asru-establishment';
+    res.locals.static.baseUrl = req.baseUrl;
+    res.locals.static.estId = req.establishment.id;
+    res.locals.static.asruUser = req.asruUser;
+    req.asruUser === 'inspectors'
+      ? res.locals.static.asru = req.establishment.asru.filter(
+        p => p.asruInspector
+      ) : res.locals.static.asru = req.establishment.asru.filter(
+        p => p.asruLicensing
+      );
```

Comment:
Can you use an `if` statement here please? Ternaries for "not assignment" break my brain.

Reply:
done